realm/realm-library/src/main/java/io/realm/RealmCache.java

```diff
@@ -342,4 +369,49 @@ private static void copyAssetFileIfNeeded(RealmConfiguration configuration) {
             }
         }
     }
+
+    /**
+     * Finds an entry for specified schema version in the array.
+     *
+     * @param array target array of schema cache.
+     * @param schemaVersion requested version of the schema.
+     * @return {@link ColumnIndices} instance for specified schema version. {@code null} if not found.
+     */
+    public static ColumnIndices find(ColumnIndices[] array, long schemaVersion) {
```

Comment:
a better name instead of `find`? Something like `findColumnIndices` or `getColumnIndices`? `find` causes trouble for me while reading the code ...